frontend/packages/noobaa-storage-plugin/src/utils.ts

```diff
@@ -1,3 +1,14 @@
 import * as _ from 'lodash';
 
 export const getPropsData = (data) => _.get(data, 'data.result[0].value[1]', null);
+
+export const getMetric = (result: PrometheusResult, metric: string): string =>
+  _.get(result, `metric.${metric}`, null);
+
+export const getValue = (result: PrometheusResult): number =>
+  Number(_.get(result, 'value[1]', null));
```

Comment:
Number(null) returns 0, I dont think this is safe

Reply:
right, so anyway 0 values are not being plotted on chart.
I intentionally put that, rather than handling `null` values. I cannot find better approach, if you could suggest something ?

Reply:
Oh I see, its in utils, I will change it later with the enhancement Pr, because i need to check which components are using it.

Reply:
I fixed it!